CONTRIBUTING.md

```diff
@@ -26,3 +26,103 @@ As described in `README.md`, we follow Agile methods and work closely with our c
 
 Hope to see you soon!
 
+### Set up your development environment with Cloud9 ( https://c9.io/ )
+
+Start by forking this repository then create new project in the Cloud9 dashboard select "Clone from URL" 
+and paste in the git URL: https://github.com/YOUR_USERNAME/osra.git and click create!
+
+Go into the ide (click the button "start editing"). First change the ruby vesion with this 
+command:
+
+$ rvm install ruby-2.2.0
+
+now you must install capybara-webkit gem
+
+$ sudo apt-get update
+
+install these package needed by Capybara-webkit to work correctly
+
+$ sudo apt-get install qt5-default libqt5webkit5-dev
+$ gem install capybara-webkit -v '1.4.1'
+
+now you can go with:
+
+$ bundle install
+
+now you have to setup the database and the permissions,
+follow these steps:
+
+$ sudo service postgresql start
+$ sudo sudo -u postgres psql
+
+now you are in postgresql console!
+
+postgres=# CREATE USER username SUPERUSER PASSWORD 'password';
+postgres=# \q
+
+setup the environment variable in Cloud9
+
+$ echo "export USERNAME=username" >> ~/.profile
+$ echo "export PASSWORD=password" >> ~/.profile
+$ source ~/.profile
+
+in the config/database.yml file change the username, password and host for all environments:
+
+  username: <%= ENV['USERNAME'] %>
+  password: <%= ENV['PASSWORD'] %>
+  host: <%= ENV['IP'] %>
+  
+then head back to postgresql console and follow these steps:
+
+$ sudo sudo -u postgres psql
+postgres=# UPDATE pg_database SET datistemplate = FALSE WHERE datname = 'template1';
```

Comment:
Are these steps necessary for Cloud9? Looks like a test of `postgres` functionality.